packages/react-reconciler/src/ReactFiberCompleteWork.new.js

```diff
@@ -649,6 +671,124 @@ function cutOffTailIfNeeded(
   }
 }
 
+function bubbleProperties(completedWork: Fiber): void {
```

Comment:
This isn't the greatest name, but neither was `resetChildLanes`. Open to suggestions.